core/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -429,7 +429,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 IndexTemplateMetaData templateMetaData = cursor.value;
                 builder.startObject(templateMetaData.name());
 
-                builder.field("template", templateMetaData.template());
+                builder.field("template", templateMetaData.patterns());
```

Comment:
OK. I think we should make this `index_patterns` or `patterns`. We're already making a breaking change here (string -> list) so let's make the output more readable while we are here. I'm unsure if this means we can't merge this to the 5.x branch or not.... I hadn't thought this through until now. It **is** is a fairly out of the way section of the code....